### ISSUE TITLE
Putting Head below Content?

### ISSUE THREAD

**Issue by douglasrcjames** [devzonetech/react-collapsible-component#1]
I am trying to put the content above the head, but when I click the head, nothing happens. When the head is above the content tag, it works. How can I get the content to below above the head?

```
<CollapsibleContent>
     Body
</CollapsibleContent>

<CollapsibleHead className="button-class">more info</CollapsibleHead>
```

**Comment by zdrsoft** [comment]
You need the content to be visible over the clickable open/hide head part?

**Comment by douglasrcjames** [comment]
Correct

**Comment by zdrsoft** [comment]
Dear @douglasrcjames ,
the requested functionality is implemented and released in version 1.3.1 of the component.

Regards,
Zdravko

**Comment by douglasrcjames** [comment]
@zdrsoft thank you!
